security/providers/oidc/src/main/java/io/helidon/security/providers/oidc/OidcProvider.java

```diff
@@ -566,7 +571,7 @@ private Principal buildPrincipal(Jwt jwt) {
         private OidcConfig oidcConfig;
         // identity propagation is disabled by default. In general we should not reuse the same token
         // for outbound calls, unless it is the same audience
-        private boolean propagate;
+        private Boolean propagate;
```

Comment:
Does the comment still apply? `identity propagation is disabled by default`

Reply:
Yes it still applies, though now it has three states (enabled/disabled/not-configured, meaning it is based on outbound targets)